analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConstructorOverridableCall.cs

```diff
@@ -55,4 +55,32 @@ public override void DoSomething()
             Console.WriteLine(this.foo.Length);
         }
     }
+
+    public class Class1
+    {
+        public virtual void DoSomething() { }
+    }
+
+    public class Class2 : Class1
+    {
+        public Class2()
+        {
+            DoSomething(); // FN
```

Comment:
Also please don't forget to open an issue for this FN as it is not C#11 related.